src/Compiler/FSharp.Compiler.Service.fsproj

```diff
@@ -361,12 +361,15 @@
     <Compile Include="Checking\CheckBasics.fs" />
     <Compile Include="Checking\CheckRecordSyntaxHelpers.fsi" />
     <Compile Include="Checking\CheckRecordSyntaxHelpers.fs" />
-    <Compile Include="Checking\CheckExpressions.fsi" />
-    <Compile Include="Checking\CheckExpressions.fs" />
+    <Compile Include="Checking\Expressions\CheckExpressionsOps.fs" />
+    <Compile Include="Checking\Expressions\CheckExpressions.fsi" />
+    <Compile Include="Checking\Expressions\CheckExpressions.fs" />
     <Compile Include="Checking\CheckPatterns.fsi" />
     <Compile Include="Checking\CheckPatterns.fs" />
-    <Compile Include="Checking\CheckComputationExpressions.fsi" />
-    <Compile Include="Checking\CheckComputationExpressions.fs" />
+    <Compile Include="Checking\Expressions\CheckComputationExpressions.fsi" />
+    <Compile Include="Checking\Expressions\CheckComputationExpressions.fs" />
+    <Compile Include="Checking\Expressions\CheckSequenceExpressions.fs" />
+    <Compile Include="Checking\Expressions\CheckArrayOrListComputedExpressions.fs" />
```

Comment:
Should these new files (`CheckExpressionsOps.fs`, `CheckSequenceExpressions.fs`, `CheckArrayOrListComputedExpressions.fs`) get signature files?

Reply:
Eventually, when settled with structure. There are only like 10-ish functions which might need signatures

Reply:
I will resolve it for now, since I have more changes coming, will do some signatures with those.